### DISCUSSION TITLE
Need help to run Nextjs with Pm2

### DISCUSSION THREAD

**Discussion by abdulbasit010** [vercel/next.js#19924] [Help]
Can anyone guide me how can I configure my NextJs app to work with Pm2 ? 
i am using this command but it throws error.

`pm2 start yarn --name "myapp" -- start`

**Comment by leo-petrucci** [comment]
That exact command works for me. What error are you getting?

**Reply by abdulbasit010** [reply]
it shows errored.

![image](https://user-images.githubusercontent.com/50239329/101356547-0b531980-38ba-11eb-8450-744e97bb8a05.png)
![image](https://user-images.githubusercontent.com/50239329/101356615-26be2480-38ba-11eb-9dfd-b9505c5e2f2e.png)

**Reply by leo-petrucci** [reply]
Looks like it's a problem with Windows? I would try running it in wsl or Linux.

**Reply by abdulbasit010** [reply]
I tried it on Linux too but facing same problem.

**Reply by leo-petrucci** [reply]
Has to be a problem with the script you're using to start it. It's likely unrelated to PM2 either way.
